apps/web/src/components/tx/ConfirmTxDetails/TxDetails.tsx

```diff
@@ -40,15 +45,17 @@ const TxDetailsRow = ({
   </Stack>
 )
 
-export const TxDetails = ({ safeTx, txData, showHashes }: TxDetailsProps) => {
+const ContentWrapper = ({ children }: { children: ReactElement | ReactElement[] }) => (
+  <Box sx={{ maxHeight: '750px', overflowY: 'auto', px: 2 }}>{children}</Box>
```

Comment:
It's way too tall now. I suggest we keep the hashes collapsed by default except for hardware wallets, and restoring 550px.

Reply:
I see your point, but I think showing all the data with the hashes provides a clearer view for verification (see below). Nonetheless, I changed it in 8088f81565c50bbda19222d8b920bd696b365acd.

![image](https://github.com/user-attachments/assets/3693f67e-5ca1-41d1-844c-8d3e54020a9f)